pom.xml

```diff
@@ -340,11 +350,15 @@
                   </module>
                   <module name="RegexpSinglelineJava">
                     <property name="format" value="junit[.]framework[.]TestCase" />
-                    <property name="message" value="Use JUnit4+ @Test annotation instead of TestCase" />
+                    <property name="message" value="Use JUnit5+ @Test annotation instead of TestCase" />
```

Comment:
This detects the JUnit3 usages, but not the JUnit4 usages. A new pattern is needed to detect JUnit4 now.

Reply:
Sound good I'll add one. This is not in the main accumulo project either (or any of the other accumulo projects) so might be a good idea to add it to the main accumulo project at least as well.

Reply:
I think we still need some JUnit4 tests in the main project, because we use PowerMock there. But, we wrote `contrib/ci/find-unapproved-junit.sh` to account for that as part of apache/accumulo#2630

Reply:
> I think we still need some JUnit4 tests in the main project, because we use PowerMock there

Oh yea I forgot about that.

I added a rule that makes sure only Junit Jupiter imports are used instead of just looking for a Junit4 `@Test` import. Take a look at [ecc0d00](https://github.com/apache/accumulo-classloaders/pull/15/commits/ecc0d0023de38838281dd9224830ad79344a1da9) and let me know what you think.